### PR TITLE
Removed bar-dependencies + added support for jnext dependencies in repo

### PR DESCRIPTION
**Pull request by jamesjhedley**
Removed bar-dependencies maven artifact + added support for jnext dependencies in repo (auth.txt)

When testing please ensure you use trunk. A recent change is required.

### CODE REVIEW

**Comment by nukulb** [comment]
When testing make sure pps API can be called 
Native API can be called
If you unzip the bar there should not be a jnext.so or pps.so in the bar file.

@jkeshavarzi - can you get another dev to test this?

**Comment by jamesjhedley** [comment]
@jeffheifetz Can you please run some tests for this pull req and see if there are any issues?

**Comment by jeffheifetz** [comment]
@derek-watson Will be helping you out with this

**Comment by derek-watson** [comment]
Tests and lint pass, test app deployed successfully!

**Comment by nukulb** [comment]
we ran into an issue, looking into it.

**Comment by nukulb** [comment]
@jkeshavarzi - ask someone else, to test its hard to be offsite and load new builds all the time.

**Comment by jamesjhedley** [comment]
@nukulb What issue did you run into?
@pagey I know I already asked you to code review this, but do you think you could test it as well?

File: Framework [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit a0c784bbe056f361527cb0484c2c5441be32635a
+Subproject commit e4c09f6a7a49b4b21b83d4ca766cf98767a35973
```
Comment (ericpearson) [inline Framework]: did we want to update the hash?

Comment (jamesjhedley) [inline Framework]: The hash it's pointing to right now is the hash for the matching Framework pull req. When the Framework pull req gets merged, we can update the hash here.

**Comment by ericpearson** [comment]
r+ and tested with no issues

**Comment by jamesjhedley** [comment]
Merged manually.

**Pull request closed without merging**